src/renderer/components/FtSearchFilters/FtSearchFilters.vue

```diff
@@ -4,12 +4,25 @@
     @click="hideSearchFilters"
   >
     <template #label="{ labelId }">
-      <h2
-        :id="labelId"
-        class="center"
-      >
-        {{ title }}
-      </h2>
+      <div class="titleContainer">
+        <h2
+          :id="labelId"
+          class="center"
+        >
+          {{ title }}
+        </h2>
+        <button
+          v-if="searchFilterValueChanged"
+          class="clearFilterButton"
+          :title="$t('Search Filters.Clear Filters')"
+          @click="clearFilters"
```

Comment:
Maybe don't make the title shift? (either it's hidden or shown as disabled by default I have no preference

https://github.com/user-attachments/assets/c25599de-6780-4a76-b507-149d8cc26801

```suggestion
          class="clearFilterButton"
          :title="$t('Search Filters.Clear Filters')"
          :style="{visibility: (searchFilterValueChanged ? 'visible' : 'hidden')}"
          @click="clearFilters"
```

Reply:
Yeah this looks much better